Projects/Server/Mobiles/Mobile.cs

```diff
@@ -3311,14 +3232,15 @@ public virtual void ProcessDelta()
             {
                 Mobile beholder;
 
-                Packet hitsPacket = null;
                 Packet statPacketTrue = null;
                 Packet statPacketFalse = null;
                 Packet hairPacket = null;
                 Packet facialhairPacket = null;
                 Packet hbpPacket = null;
                 Packet hbyPacket = null;
 
+                Span<byte> statsPacket = stackalloc byte[OutgoingMobilePackets.MobileStatsPacketLength];
```

Comment:
I could probably reuse the struct above, but I would need a way to detect if it was already written. This is fine instead.

Reply:
One option would be to do:
```cs
statsPacket[0] = 0;
```
Instead of this line. Which, depending on the size of the stackalloc, and optimizations, may not even matter.